x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/evaluate_panel.tsx

```diff
@@ -76,6 +83,25 @@ export const EvaluatePanel: FC<Props> = ({ jobConfig, jobStatus, searchQuery })
   }) => {
     setIsLoading(true);
 
+    try {
+      const indexPatternId = getIndexPatternIdFromName(sourceIndex) || sourceIndex;
+      const indexPattern: IndexPattern = await kibanaContext.indexPatterns.get(indexPatternId);
+
+      if (indexPattern !== undefined) {
+        await newJobCapsService.initializeFromIndexPattern(indexPattern);
+        // Check dependent_variable field type to see if .keyword suffix is required for evaluate endpoint
+        const { fields } = newJobCapsService;
+        const depVarFieldType = fields.find((field: any) => field.name === dependentVariable)?.type;
```

Comment:
By saying this field is `any` you're losing the type information that you get from free from the `fields` array.
there is no need to add a type here at all as it knows it's a `Field[]`

Reply:
Good point - removed in 0a79e2a39475beff24b51a40e78f6d6d3e806cca